test-requirements.txt

```diff
@@ -15,8 +15,8 @@ typeguard>=2.5
 paramiko
 dill
 tblib
-ipyparallel
 psutil>=5.5.1
 globus-sdk
 twine
 wheel
+pyzmq>=17.1.2
```

Comment:
what is this for?

Reply:
pyzmq was a dependency in ipp, and we didn't have it in requirements till now. Now that IPP is completely removed, we have to specify this. I just noticed that ipykernel is still in there, and I'm removing it.

Reply:
it's a duplicate with `requirements.txt`, though - it shouldn't need to be in test-requirements if it is also a core parsl requirement.